templates/default/opencast-setenv.erb

```diff
@@ -31,8 +31,11 @@
 # export KARAF_BASE        # Karaf base folder
 # export KARAF_ETC         # Karaf etc  folder
 # export KARAF_OPTS        # Additional Karaf options
-# export KARAF_DEBUG       # Enable debug mode
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
+#
+# Debug options
+# export KARAF_DEBUG       # Enable debug mode
+# export JAVA_DEBUG_PORT   # Set debug port (defaults to 5005)
```

Comment:
Is adding option to change the default debug port to allow the service stop & and start to do the jdwp check to see if the service is running or stopped?

Reply:
This came from upstream so I'm not sure.